cypress/support/helpers/topicTagsTest.js

```diff
@@ -1,15 +1,52 @@
 import getDataUrl from './getDataUrl';
 import visitPage from './visitPage';
 
-export default () => {
+const getPageData = (url, service, variant, pageType) => {
+  const env = Cypress.env('APP_ENV');
+  let pageBody;
+
+  const isBffFetch = pageType === 'articles';
+
+  if (!isBffFetch) {
+    pageBody = cy.request(getDataUrl(url)).then(({ body }) => body);
+  } else {
+    const articleId =
+      Cypress.env('currentPath').match(/(c[a-zA-Z0-9]{10}o)/)?.[1];
+
+    const bffUrl = `https://web-cdn.${
+      env === 'live' ? '' : `${env}.`
+    }api.bbci.co.uk/fd/simorgh-bff?pageType=article&id=${articleId}&service=${service}${
+      variant ? `&variant=${variant}` : ''
+    }`;
```

Comment:
Should we try and hide this in an environment variable instead of exposing this path?

Reply:
We have `SIMORGH_MOST_READ_CDN_URL` which is the same endpoint confusingly. Would probably be better to have it as a Cosmos variable, but I'm not sure how well Cypress interacts with Cosmos to get those.

Reply:
Ah, no worries, let's leave this as is.